chanzuckerberg/czgenepi
# FE render link

LIVE: https://thuang-auspice-secure-2-frontend.dev.genepi.czi.technology/data/phylogenetic_trees

### Description

1. Update FE to fetch presigned URL from the private endpoint, and then encode the presigned URL + construct `nextstrain.org` URL
2. Update BE to return a JSON `{ "url": PRESIGNED_URL }`

#### Issue
N/A

### Test plan

1. Load https://thuang-auspice-secure-2-frontend.dev.genepi.czi.technology/data/phylogenetic_trees
2. Click on any tree and see the modal
3. Click on confirm and see the tree on Nextstrain!
